slices/fontconfig-config.yaml

```diff
@@ -0,0 +1,83 @@
+package: fontconfig-config
+
+essential:
+  - fontconfig-config_copyright
+
+slices:
+  config:
+    essential:
+      - fonts-dejavu-core_config
+      - fonts-dejavu-core_fonts
+    contents:
+      /etc/fonts/conf.d/10-hinting-slight.conf:
+        symlink: /usr/share/fontconfig/conf.avail/10-hinting-slight.conf
```

Comment:
you don't need to create the symlink. this file is already a symlink in the deb pkg, so Chisel will extract it as such. (same for the remaining files below)